### ISSUE TITLE
Unable to contact servers / coordinates throw out 

### ISSUE THREAD

**Issue by Aumis** [NECROBOTIO/NecroBot#5453]
## Stop! Before you create this issue (you can delete this section when opening the issue):
1. Have you validated that your config/auth.json and config/config.json is valid JSON? Use http://jsonlint.com/ to check.
2. Have you searched to see if there are other issues for the same issue? If so, comment on that issue instead.
3. If your question is referring to how to set up or use the bot, please join our discord instead of posting an issue: https://discord.gg/VXKxNFr

### Expected Behavior
Last night before the update my coordinates was good  login details right was farming like mad. 
51.4586302
0.7169628

### Actual Behavior
Then i found this morning the bot had an issue saying about servers may be off line but at the same time
it was throwing my coordinates out the window by adding extra numbers on the end of the position i have chosen.
![capture](https://cloud.githubusercontent.com/assets/20953280/17618215/41bba724-6076-11e6-8b33-9f32bf6d48cb.PNG)

### Your config.json
```
your config here
```
{
  "TranslationLanguageCode": "en",
  "AutoUpdate": true,
  "TransferConfigAndAuthOnUpdate": true,
  "UseWebsocket": false,
  "WebSocketPort": 14251,
  "StartupWelcomeDelay": false,
  "UseTelegramAPI": false,
  "TelegramAPIKey": null,
  "AmountOfPokemonToDisplayOnStart": 10,
  "DetailedCountsBeforeRecycling": true,
  "MaxBerriesToUsePerPokemon": 3,
  "CatchPokemon": true,
  "AutomaticallyLevelUpPokemon": false,
  "OnlyUpgradeFavorites": true,
  "UseLevelUpList": true,
  "AmountOfTimesToUpgradeLoop": 5,
  "GetMinStarDustForLevelUp": 5000,
  "LevelUpByCPorIv": "iv",
  "UpgradePokemonCpMinimum": 1000.0,
  "UpgradePokemonIvMinimum": 95.0,
  "UpgradePokemonMinimumStatsOperator": "and",
  "DisableHumanWalking": false,
  "DefaultLatitude": 51.384406,
  "DefaultLongitude": 0.5241843,
  "WalkingSpeedInKilometerPerHour": 19.0,
  "MaxSpawnLocationOffset": 10,
  "FastSoftBanBypass": false,
  "DelayBetweenPlayerActions": 500,
  "DelayBetweenPokemonCatch": 100,
  "DumpPokemonStats": false,
  "EvolveAboveIvValue": 95.0,
  "EvolveAllPokemonAboveIv": false,
  "EvolveAllPokemonWithEnoughCandy": true,
  "EvolveKeptPokemonsAtStorageUsagePercentage": 90.0,
  "KeepPokemonsThatCanEvolve": false,
  "KeepMinCp": 1250,
  "KeepMinIvPercentage": 90.0,
  "KeepMinLvl": 6,
  "KeepMinOperator": "or",
  "UseKeepMinLvl": false,
  "PrioritizeIvOverCp": false,
  "KeepMinDuplicatePokemon": 0,
  "UseGpxPathing": false,
  "GpxFile": "GPXPath.GPX",
  "VerboseRecycling": true,
  "RecycleInventoryAtUsagePercentage": 90.0,
  "RandomizeRecycle": false,
  "RandomRecycleValue": 5,
  "DelayBetweenRecycleActions": false,
  "UseEggIncubators": true,
  "minEggKmForLimitedIncubators": 2,
  "UseLuckyEggConstantly": false,
  "UseLuckyEggsMinPokemonAmount": 30,
  "UseLuckyEggsWhileEvolving": true,
  "UseIncenseConstantly": true,
  "UseBerriesMinCp": 1000,
  "UseBerriesMinIv": 90.0,
  "UseBerriesBelowCatchProbability": 0.2,
  "UseBerriesOperator": "or",
  "UseSnipeLocationServer": false,
  "SnipeLocationServer": "localhost",
  "SnipeLocationServerPort": 16969,
  "GetSniperInfoFromPokezz": true,
  "GetOnlyVerifiedSniperInfoFromPokezz": true,
  "GetSniperInfoFromPokeSnipers": true,
  "GetSniperInfoFromPokeWatchers": true,
  "SnipeWithSkiplagged": true,
  "MinPokeballsToSnipe": 20,
  "MinPokeballsWhileSnipe": 0,
  "MinDelayBetweenSnipes": 60000,
  "SnipingScanOffset": 0.005,
  "SnipeAtPokestops": false,
  "SnipeIgnoreUnknownIv": false,
  "UseTransferIvForSnipe": false,
  "SnipePokemonNotInPokedex": false,
  "RenamePokemon": false,
  "RenameOnlyAboveIv": true,
  "RenameTemplate": "{1}_{0}",
  "MaxPokeballsPerPokemon": 6,
  "MaxTravelDistanceInMeters": 20000,
  "TotalAmountOfPokeballsToKeep": 120,
  "TotalAmountOfPotionsToKeep": 80,
  "TotalAmountOfRevivesToKeep": 60,
  "TotalAmountOfBerriesToKeep": 50,
  "UseGreatBallAboveCp": 1000,
  "UseUltraBallAboveCp": 1250,
  "UseMasterBallAboveCp": 1500,
  "UseGreatBallAboveIv": 85.0,
  "UseUltraBallAboveIv": 95.0,
  "UseGreatBallBelowCatchProbability": 0.2,
  "UseUltraBallBelowCatchProbability": 0.1,
  "UseMasterBallBelowCatchProbability": 0.05,
  "EnableHumanizedThrows": false,
  "NiceThrowChance": 40,
  "GreatThrowChance": 30,
  "ExcellentThrowChance": 10,
  "CurveThrowChance": 90,
  "ForceGreatThrowOverIv": 90.0,
  "ForceExcellentThrowOverIv": 95.0,
  "ForceGreatThrowOverCp": 1000,
  "ForceExcellentThrowOverCp": 1500,
  "TransferWeakPokemon": false,
  "TransferDuplicatePokemon": true,
  "TransferDuplicatePokemonOnCapture": true,
  "FavoriteMinIvPercentage": 95.0,
  "AutoFavoritePokemon": false,
  "UsePokemonToNotCatchFilter": false,
  "UsePokemonSniperFilterOnly": false,
  "ItemRecycleFilter": [
    {
      "Key": "itemUnknown",
      "Value": 0
    },
    {
      "Key": "itemLuckyEgg",
      "Value": 200
    },
    {
      "Key": "itemIncenseOrdinary",
      "Value": 100
    },
    {
      "Key": "itemIncenseSpicy",
      "Value": 100
    },
    {
      "Key": "itemIncenseCool",
      "Value": 100
    },
    {
      "Key": "itemIncenseFloral",
      "Value": 100
    },
    {
      "Key": "itemTroyDisk",
      "Value": 100
    },
    {
      "Key": "itemXAttack",
      "Value": 100
    },
    {
      "Key": "itemXDefense",
      "Value": 100
    },
    {
      "Key": "itemXMiracle",
      "Value": 100
    },
    {
      "Key": "itemSpecialCamera",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasicUnlimited",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasic",
      "Value": 100
    },
    {
      "Key": "itemPokemonStorageUpgrade",
      "Value": 100
    },
    {
      "Key": "itemItemStorageUpgrade",
      "Value": 100
    }
  ],
  "PokemonsNotToTransfer": [
    "venusaur",
    "charizard",
    "blastoise",
    "clefable",
    "muk",
    "chansey",
    "gyarados",
    "ditto",
    "snorlax",
    "articuno",
    "zapdos",
    "moltres",
    "dragonite",
    "mewtwo",
    "mew"
  ],
  "PokemonsToEvolve": [
    "caterpie",
    "weedle",
    "pidgey",
    "rattata"
  ],
  "PokemonsToLevelUp": [
    "caterpie",
    "weedle",
    "pidgey",
    "rattata",
    "spearow",
    "zubat",
    "doduo"
  ],
  "PokemonsToIgnore": [
    "caterpie",
    "weedle",
    "pidgey",
    "rattata",
    "spearow",
    "zubat",
    "doduo"
  ],
  "PokemonsTransferFilter": {
    "Golduck": {
      "KeepMinCp": 1800,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Farfetchd": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Krabby": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Kangaskhan": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 60.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Horsea": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Staryu": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "MrMime": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 40.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Scyther": {
      "KeepMinCp": 1800,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Jynx": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Electabuzz": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Magmar": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Pinsir": {
      "KeepMinCp": 1800,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Tauros": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Magikarp": {
      "KeepMinCp": 200,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Gyarados": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Lapras": {
      "KeepMinCp": 1800,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Eevee": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Vaporeon": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Jolteon": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Flareon": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Porygon": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 60.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Snorlax": {
      "KeepMinCp": 2600,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Dragonite": {
      "KeepMinCp": 2600,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    }
  },
  "PokemonToSnipe": {
    "Locations": [
      {
        "Latitude": 38.556807486461118,
        "Longitude": -121.2383794784546
      },
      {
        "Latitude": -33.859019,
        "Longitude": 151.213098
      },
      {
        "Latitude": 47.5014969,
        "Longitude": -122.0959568
      },
      {
        "Latitude": 51.5025343,
        "Longitude": -0.2055027
      }
    ],
    "Pokemon": [
      "venusaur",
      "charizard",
      "blastoise",
      "beedrill",
      "raichu",
      "sandslash",
      "nidoking",
      "nidoqueen",
      "clefable",
      "ninetales",
      "golbat",
      "vileplume",
      "golduck",
      "primeape",
      "arcanine",
      "poliwrath",
      "alakazam",
      "machamp",
      "golem",
      "rapidash",
      "slowbro",
      "farfetchd",
      "muk",
      "cloyster",
      "gengar",
      "exeggutor",
      "marowak",
      "hitmonchan",
      "lickitung",
      "rhydon",
      "chansey",
      "kangaskhan",
      "starmie",
      "mrMime",
      "scyther",
      "magmar",
      "electabuzz",
      "jynx",
      "gyarados",
      "lapras",
      "ditto",
      "vaporeon",
      "jolteon",
      "flareon",
      "porygon",
      "kabutops",
      "aerodactyl",
      "snorlax",
      "articuno",
      "zapdos",
      "moltres",
      "dragonite",
      "mewtwo",
      "mew"
    ]
  },
  "PokemonToUseMasterball": [
    "articuno",
    "zapdos",
    "moltres",
    "mew",
    "mewtwo"
  ]
}

### Other Information
Version: v0.8.6
Windows Version: Windows 10
What is affected (catch, transfer, evolve, recycle, snipe, powerup/levelup, other):

**Comment by CrispyBaconUk** [comment]
Yes certainly looking like a ban of some sort, i have exactly the same issue, however if i switch accounts all is fine. It does look like a bug on the config tool however it is still in experimental beta so kinda to be expected

**Comment by Aumis** [comment]
ok so i tried logging in  manually  and here is what i got 
![capture2](https://cloud.githubusercontent.com/assets/20953280/17618829/bebd7664-6079-11e6-9cf7-5013e3f6c541.PNG)

**Comment by headhunter1215** [comment]
@Aumis  i meant on your phone, to me it looks like some sort of ban or if we're really lucky some sort of soft ban or server fail.

**Comment by Aumis** [comment]
Logged into the app on my iphone 5c and get 
Failed to get game data from the server

**Comment by headhunter1215** [comment]
@Aumis  yes i got the same for 2 of my account's that used necro bot but can use a other one that didn't... 
doesn't look to good, let's hope we're really lucky and it's some kind of new "soft ban" against bots but worse case scenario we are perm banned.
